api/agentic/v1alpha1/doc.go

```diff
@@ -0,0 +1,18 @@
+/*
+Copyright 2025.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// +groupName=agentic.networking.x-k8s.io
+package v1alpha1
```

Comment:
lets change it to v1prototype1 here, the folders and all the rest. No need for agentic middle layer